crates/bitwarden/src/crypto/aes_ops.rs

```diff
@@ -12,22 +12,23 @@ use crate::{
 
 pub fn decrypt_aes256(
     iv: &[u8; 16],
-    mac: &[u8; 32],
+    mac: Option<&[u8; 32]>,
```

Comment:
It would be nice to ensure the mac and mac-key is always set or not set. Do you think we should have two methods for this instead? Or some way to wrap both mac and key in an option

Reply:
I've made it that way to try to change the function definition too much, but it's definitely better as two functions and it's not used in that many places